mautic/mautic
# [UX] Make "Add widget" the main action on the dashboard

| Q                                      | A
| -------------------------------------- | ---
| Bug fix? (use the a.b branch)          | [ ]
| New feature/enhancement? (use the a.x branch)      | [ ]
| Deprecations?                          | [ ]
| BC breaks? (use the c.x branch)        | [ ]
| Automated tests included?              | [ ] 
| Related user documentation PR URL      | mautic/mautic-documentation#... 
| Related developer documentation PR URL | mautic/developer-documentation#... 
| Issue(s) addressed                     | Fixes #... 



#### Description:
The primary action button on a page should represent the most common action used by users.
This change makes the primary button on the page to add a widget, as well as refining the text of the save button and reducing the verbosity of the import action.

Discussed on: https://mautic.slack.com/archives/C024XBCQS3Z/p1712174455596019



#### Steps to test this PR:


1. Open this PR on Gitpod or pull down for testing locally (see docs on testing PRs [here](https://contribute.mautic.org/contributing-to-mautic/tester))
2. Open the dashboard
3. See the new page action
4. Click to open the dropdown and see new labels for already existing features

![image](https://github.com/mautic/mautic/assets/116097999/a1d87cb5-9034-446a-a272-3bb25d69ad1c)
